deploy/Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN docker-credential-gcr configure-docker
 # Get Amazon ECR credential helper
 RUN go get -u github.com/awslabs/amazon-ecr-credential-helper/ecr-login/cli/docker-credential-ecr-login
 RUN make -C /go/src/github.com/awslabs/amazon-ecr-credential-helper linux-amd64
+# ACR docker credential helper
+ADD https://aadacr.blob.core.windows.net/acr-docker-credential-helper/docker-credential-acr-linux-amd64.tar.gz /usr/local/bin
```

Comment:
Can we lock this to a specific version of the credential helper?

Reply:
As far as i can see this tool does not have releases / version numbers yet.

Reply:
Ya that's what I see too, bummer. Hopefully we won't have too many problems with updates